frontend/apps/web/components/site-header/SiteHeader.tsx

```diff
@@ -31,7 +31,8 @@ export default function SiteHeader(): ReactElement {
         <MainNav />
         <MobileNav />
         <div className="flex flex-1 justify-end items-center space-x-2">
-          {systemAppConfig.isNeosyncCloud && <AccountsRecordProgress />}
+          {systemAppConfig.isNeosyncCloud &&
+            systemAppConfig.isStripeEnabled && <AccountsRecordProgress />}
```

Comment:
This is better but not great. We should really be checking the account status and showing only if they have a max allowed records set. But this is good enough for now.

Reply:
We can update the component to instead pull from the account status which has all of this information